doc/develop/getting_started/index.rst

```diff
@@ -487,6 +487,13 @@ Zephyr applications.
 It also contains additional host tools, such as custom QEMU and OpenOCD builds
 that are used to emulate, flash and debug Zephyr applications.
 
+.. note::
+
+    Although the Zephyr SDK is recommended for new users, alternative
+    toolchains are supported. Users may check the list of
+    :ref:`alternate toolchains <toolchains>` to see if an existing toolchain
+    they have installed is supported by Zephyr.
```

Comment:
While I see no problem on mentioning this here, I'd not make it a note. I think that new users will have a better/more uniform experience if they use the SDK because it also includes some host tools.